docs/COMMANDS.md

```diff
@@ -0,0 +1,141 @@
+# Commands
+GlueStick comes with several commands to help you get started. You can full list of commands available on gluestick below:
+
+### `gluestick new`
+
+Takes care of generating new project
+
+```bash
+gluestick new <YOUR_APPLICATION_NAME>
+```
+
+Available options:
+
+* `-d, --dev <path>` - Relative path to development version of gluestick
+* `-y, --yarn` - Use yarn to perform installations
+
+### `gluestick generate`
+
+Generates a new entity from given template
+
+```bash
+gluestick generate <ENTITY_TYPE> <ENTITY_NAME>
+```
+
+Available options:
+
+* `-E --entry-point <entryPoint>` - Entry point for generated files
+* `-F, --functional` - Generate stateless functional component
+* `-O, --gen-options <value>` - Options to pass to the generator
```

Comment:
There should be a line somwhere here that points to `GENERATORS.md`. Another thing is... I am not sure why we follow that name convention, I would do it only for `README`. The others can be like `Generators.md` (that is what I see in other repos).